utils/logger.py

```diff
@@ -54,6 +55,12 @@ def save_logs_to_bucket(bucket, bucket_output_path, output_path, now, batch_metr
 
 
 def dict2path(my_dict, path=None):
+    """
+    TODO
+    @param my_dict:
+    @param path:
+    @return:
+    """
```

Comment:
We should probably fill those?

Reply:
I agree! I added a TODO to make sure we don't forget. I think you were the one to add them at PR #208 . Do you think you can add docstrings for these functions?